src/python/pants/backend/python/dependency_inference/import_parser.py

```diff
@@ -59,6 +59,18 @@ def visit_ImportFrom(self, node):
         for alias in node.names:
             self.explicit_imports.add("{}.{}".format(abs_module, alias.name))
 
+    def visit_Call(self, node):
+      # Handle __import__("string_literal").  This is commonly used in __init__.py files,
+      # to explicitly mark namespace packages.
+      if (
+          isinstance(node.func, ast.Name)
+          and node.func.id == "__import__"
+          and len(node.args) == 1
```

Comment:
The only problematic argument I can see is `level`, which if non-zero would require more logic to handle correctly. Is that the impetus for only handling the single-argument form?

Reply:
I didn't single out `level` specifically, I just wanted to cover the common case of `__import__("pkg_resources").declare_namespace(__name__)`

I'm OK if we don't cover more complicated cases.